views/pbxsManager.html

```diff
@@ -22,14 +22,21 @@
 
 			<div id="unassigned_actions" class="row-fluid">
 				<div class="span6 link-box assign">
-					<div class="span12"><i class="fa fa-check-square-o"></i>{{ i18n.assign }}</div>
+					<div class="span12"><i class="fa fa-plus-circle"></i>{{ i18n.assign }}</div>
 				</div>
 
 				<div class="span6 link-box delete">
 					<div class="span12"><i class="fa fa-trash-o"></i>{{ i18n.delete }}</div>
 				</div>
 			</div>
 
+			<div id="unassigned_select_all" class="row-fluid">
+				<div class="span12 text-center link-box">
+					<i class="fa fa-check-square-o unassigned-select-all-icon"></i> <span class="unassigned-select-all-label">{{ i18n.select_all }}</span>
```

Comment:
In general, we try to use CSS instead of spaces in templates in order to style elements. Here, adding a `margin` on the icon should help you achieve the expected result.

I recommend you look at the CSS rules of `#unassigned_actions` and try to merge it with the ones you added for `#unassigned_select_all`.

Also, we also try to keep each element on one line so it is more readable (especially on git diffs). Obviously this old template does not follow that rule but you can clean it up locally:
```suggestion
					<i class="fa fa-check-square-o unassigned-select-all-icon"></i>
					<span class="unassigned-select-all-label">
						{{ i18n.select_all }}
					</span>
```
This way, if you make a change latter to the icon for example, it will be easier to check what changed.

Reply:
done